guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -20,12 +20,14 @@ An internal label that matches the value is also created and cannot be changed a
 If you do not specify a value, an organization called *Default Organization* with the label *Default_Organization* is created.
 You can rename the organization name but not the label.
 
-* By default, all configuration files configured by the installer are managed by Puppet.
-When `{foreman-installer}` runs, it overwrites any manual changes to the Puppet managed files with the initial values.
+* By default, all configuration files configured by the installer are managed.
+When `{foreman-installer}` runs, it overwrites any manual changes managed files with the initial values.
+ifdef::foreman-el,foreman-deb[]
 By default, {ProjectServer} is installed with the Puppet agent running as a service.
 If required, you can disable Puppet agent on {ProjectServer} using the `--puppet-runmode=none` option.
+endif::[]
 
-* If you want to manage DNS files and DHCP files manually, use the `--foreman-proxy-dns-managed=false` and `--foreman-proxy-dhcp-managed=false` options so that Puppet does not manage the files related to the respective services.
+* If you want to manage DNS files and DHCP files manually, use the `--foreman-proxy-dns-managed=false` and `--foreman-proxy-dhcp-managed=false` options so that the installer does not manage the files related to the respective services.
```

Comment:
I was debating if this needs to be pulled into the section started on line 23. On the other hand, I believe we should drop it. We don't have any instructions here to set up DHCP and DNS so isn't it confusing to have this as the initial instructions? I wanted to leave this as a follow up.

Reply:
I'd agree, this does seem like a "big" part of the deployment to try to consider here without any additional context that would be better off in the sections on dns / dhcp.

Reply:
I agree to move it. Not sure about removing it entirely.

Reply:
Let's have that discussion in https://github.com/theforeman/foreman-documentation/pull/2694.